spec/lib/network/abstract_server_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'network/abstract_server'
+
+describe Network::AbstractServer do
+    context 'initialize' do
+        it 'returns true if AbstractServer inherits AbstractClient' do
+            expected = true
+
+            expect(expected).to eq Network::AbstractServer.ancestors.include?(Network::AbstractClient)
```

Comment:
I think this works if you are testing for true

```suggestion
            expect(Network::AbstractServer.ancestors.include?(Network::AbstractClient)).to be true
```